trusted_os/rpc.go

```diff
@@ -156,6 +158,26 @@ func (r *RPC) Read(xfer rpc.Read, out *[]byte) (err error) {
 	return
 }
 
+// ReadIdentityCounterStorage reads data from the storage media sector
+// containing the witness identity counter.
+func (r *RPC) ReadIdentityCounterStorage(_ any, counter *uint32) error {
```

Comment:
An alternative to adding a new RPC could be to add the counter to the `Status` RPC (the applet calls this just after it starts), wdyt?

Reply:
First part done in https://github.com/transparency-dev/armored-witness-os/pull/155. Also made the changes here after merging this branch with that one.